### PR TITLE
UNITY-9029 isInWishlist attribute

### PR DESCRIPTION
**Pull request by mhoehn**

### CODE REVIEW

File: _raml/apps/products/examples/get_product_productid_crossselling_response.json [hunk 1]
```diff
@@ -230,6 +230,7 @@
         "refUnit": "piece",
         "refContentAmount": 1,
         "priceQuantity": 1,
+        "isInWishlist": false,
```
Comment (dkluever22) [inline _raml/apps/products/examples/get_product_productid_crossselling_response.json:233]: The placement of the attribute is a bit odd. I would rather place it between `searchKeywords` and `links`. What do you think?

File: _raml/apps/products/examples/get_product_productid_crossselling_response.json [hunk 2]
```diff
@@ -369,6 +370,7 @@
...
```
Comment (dkluever22) [inline _raml/apps/products/examples/get_product_productid_crossselling_response.json:373]: The placement of the attribute is a bit odd. I would rather place it between `searchKeywords` and `links`. What do you think?

**Review by dkluever22 — COMMENTED** [review]
@mhoehn Please take a look at my comments and suggestions :)
Furthermore, I don't know what is exactly triggers the new attribute to be added but I feel like it is missing in some places.
In my opinion it is missing in the following examples:
- _patch_product_productid_crossselling_response.json_
- _patch_product_productid_request.json_
- _put_product_productid_crossselling_response.json_
In addition I would think that if it has been added to _patch_product_productid_response.json_ this results in the need to adjust the _product.raml_ as well, right? By adding `/isInWishlist` to the `PATCH` endpoint?

Could you please have another look and check, if this might be right?

Regarding the `carts` resource I'm super unsure. Maybe you could double-check it as well 😇 🙏 ?!

Many thanks in advance, Micha!

**Comment by mhoehn** [comment]
DONE:  Please take a look at my comments and suggestions :) 

Furthermore, I don't know what is exactly triggers the new attribute to be added but I feel like it is missing in some places.
In my opinion it is missing in the following examples:

    patch_product_productid_crossselling_response.json.   >>> added 
    patch_product_productid_request.json.                           >>> not added (we can not use this attribute for patching)
    put_product_productid_crossselling_response.json.      >>> added 

    In addition I would think that if it has been added to patch_product_productid_response.json this results in the need to adjust the product.raml as well, right? By adding /isInWishlist to the PATCH endpoint?

>>>We can not patch a product using this attibute but the response returns also this attribute bc. we return the whole product.
So there is no deed for adding /isInWishlist to the PATCH endpoint.

**Review by dkluever22 — APPROVED** [review]

**Pull request merged**